### PR TITLE
fix(tail): add support for negative indexing

### PR DESCRIPTION
**Pull request by ycd**
closes: https://github.com/uutils/coreutils/issues/1860

### CODE REVIEW

**Comment by sylvestre** [comment]
Please add a test :)

File: tests/by-util/test_tail.rs [hunk 1]
```diff
@@ -329,3 +329,12 @@ fn test_multiple_input_quiet_flag_overrides_verbose_flag_for_suppressing_headers
         .run()
         .stdout_is_fixture("foobar_multiple_quiet.expected");
 }
+
+#[test]
+fn test_negative_indexing() {
```
Comment (sylvestre) [inline tests/by-util/test_tail.rs:334]: please do the same for -c :)

Comment (ycd) [reply tests/by-util/test_tail.rs:334]: Sure!

**Comment by ycd** [comment]
Hmmm CI failed for `i686` but it looks unrelated :thinking: 

```
failures:
    test_chown::test_passgrp::test_gid2grp
```

**Comment by sylvestre** [comment]
Yeah, we have some intermittent test issues!

**Pull request merged**